### PR TITLE
Lesson updates

### PR DESCRIPTION
**Pull request by kkiesling**
Added in the rest of the lessons, or changed them to be based on the python-novice-gapmeinder lessons as necessary. I still need to probably tweak content, but I will make a PR for each lesson for tweeks.

### CODE REVIEW

File: _episodes/04-lists.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
+---
+title: "Lists"
+teaching: 10
+exercises: 10
```
Comment (pshriwise) [inline _episodes/04-lists.md]: Are we going to allot time for exercises? It looks like time is going to be tight if we do.

**Review by pshriwise — COMMENTED** [review]
One small comment. Looks good otherwise.

File: _episodes/04-lists.md (hunk 1 above)
Comment (kkiesling) [reply _episodes/04-lists.md]: nope - that is a mistake. I will fix that in the Lists and Dict/Tuples lessons.

**Review by pshriwise — APPROVED** [review]
Looks good! Nice to see it start coming together. Thanks @kkiesling 👍

**Pull request merged**